script/release-pr

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -e
+
+args=${@:1}
+package=primer-css
+npm_tag=pr
+# the commit status context
+context=npm/publish/${npm_tag}
+
+echo "🐦  Publishing PR release to dist-tag '${npm_tag}'..."
+lerna publish --npm-tag=${npm_tag} --canary --no-git ${args}
```

Comment:
Double checking here, so we're still doing a `--canary` publish (because it does what we need) but we changed the dist tag to `release-pr` because it's a better description of the release we're publishing? It sorta feels odd but I guess that made sense to lerna folks and naming things is hard :woman_shrugging: